runtime/rococo/src/lib.rs

```diff
@@ -81,9 +81,6 @@ include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
 
 pub mod asset_config;
 pub mod constants;
-// rococo already upgraded to runtime 9130
-// TODO: delete the commented lines and rename the migration file
-// pub mod migration;
```

Comment:
line `// migration::MigrateAtStakeAutoCompound<Runtime>,` needs to be removed too